padl/dumptools/symfinder.py

```diff
@@ -342,7 +339,18 @@ def deparse(self):
             source = self._result._source
         except AttributeError:
             source = self.source
-        return f'{self.var_name} = {ast_utils.get_source_segment(source, self._result.value)}'
+        return ast_utils.get_source_segment(source, self._result)
+
+
+class _SetAttribute(ast.NodeVisitor):
```

Comment:
I know this is a simple class, but I would appreciate a short docstring explaining what's going on.

Reply:
I'll add